pack/map.h

```diff
@@ -339,6 +357,20 @@ const Attribute& Map<T>::get(const std::string& key) const
     throw std::out_of_range("Key " + key + " was not found");
```

Comment:
I'm not sure if we need to check and throw exception in this case, because std::map - Returns a reference to the value that is mapped to a key equivalent to key, performing an insertion if such key does not already exist.